connexion/apps/flask_app.py

```diff
@@ -38,6 +42,14 @@ def create_app(self):
         app.url_map.converters['int'] = IntegerConverter
         return app
 
+    def apply_middleware(self):
```

Comment:
It feels a bit weird to have this as a public function but at the same time it's not straightforward to apply any other middleware. Now, it's a function that doesn't need to be exposed as it's only done at instance creation?

I would propose to make it either a private function, or to allow for an argument for the entire middleware stack or to extend the middleware stack.